i18n/qqq.json

```diff
@@ -13,7 +14,33 @@
 	},
 	"action-managewiki": "Button to manage the settings of a wiki.",
 	"apihelp-query+wikiconfig-summary": "{{doc-apihelp-summary|wikiconfig}}",
-	"logentry-managewiki-settings": "Used as log message.",
+	"apihelp-query+wikiconfig-example-1": "{{doc-apihelp-example|wikiconfig}}",
+        "apihelp-query+wikiconfig-param-prop": "{{doc-apihelp-param|wikiconfig|prop}}",
```

Comment:
Not sure why the indentation is showing up like this. I am sure I did it correctly.

Reply:
You have spaces here not tabs. Should be tabs

Reply:
Hmm... Let me try...

Reply:
GitHub does not like it. I will test this once I add more messages.